src/index.js

```diff
@@ -21,6 +21,12 @@ export class Animate extends React.Component {
     this.animating = false;
   }
 
+  componentWillMount() {
+    if (this.props.updateOnMount) {
```

Comment:
I'm not sure I understand the point of this. I haven't looked at this code in a while, but I see that `triggerAnimations` is called inside `componentDidMount`. If the props set on the element are different from the ones set on `Animate`, that will trigger an animation on mount.

Does that not work for your use case? What's an example that `updateOnMount` solves? Can it be done inside `componentDidMount`, where `this.node` is guaranteed to exist?